notebooks/224-distilbert-Sequence-Classification/224-distilbert-Sequence-Classification.ipynb

```diff
@@ -0,0 +1,416 @@
+      "\t- Precision of IR: \tFP32\n",
+      "\t- Enable fusing: \tTrue\n",
+      "\t- User transformations: \tNot specified\n",
+      "\t- Reverse input channels: \tFalse\n",
+      "\t- Enable IR generation for fixed input shape: \tFalse\n",
+      "\t- Use the transformations config file: \tNone\n",
+      "Advanced parameters:\n",
+      "\t- Force the usage of legacy Frontend of Model Optimizer for model conversion into IR: \tFalse\n",
+      "\t- Force the usage of new Frontend of Model Optimizer for model conversion into IR: \tFalse\n",
+      "OpenVINO runtime found in: \tc:\\users\\arunimac\\appdata\\local\\programs\\python\\python38\\lib\\site-packages\\openvino\n",
+      "OpenVINO runtime version: \t2022.2.0-7690-940e927a22b-refs/pull/1296/head\n",
+      "Model Optimizer version: \t2022.2.0-7690-940e927a22b-refs/pull/1296/head\n",
+      "[ SUCCESS ] Generated IR version 11 model.\n",
+      "[ SUCCESS ] XML file: C:\\Users\\arunimac\\OneDrive - Intel Corporation\\Documents\\Projects\\OpenVINO Contrib\\224-distilbert-Sequence-Classification\\model\\distilbert-base-uncased-finetuned-sst-2-english.xml\n",
+      "[ SUCCESS ] BIN file: C:\\Users\\arunimac\\OneDrive - Intel Corporation\\Documents\\Projects\\OpenVINO Contrib\\224-distilbert-Sequence-Classification\\model\\distilbert-base-uncased-finetuned-sst-2-english.bin\n",
+      "[ SUCCESS ] Total execution time: 4.55 seconds. \n",
+      "[ INFO ] The model was converted to IR v11, the latest model format that corresponds to the source DL framework input/output format. While IR v11 is backwards compatible with OpenVINO Inference Engine API v1.0, please use API v2.0 (as of 2022.1) to take advantage of the latest improvements in IR v11.\n",
+      "Find more information about API v2.0 and IR v11 at https://docs.openvino.ai\n"
+     ]
+    }
+   ],
+   "source": [
+    "onnx_model_path = 'model.onnx'\n",
+    "MODEL_DIR = f\"model/\"\n",
+    "os.makedirs(MODEL_DIR, exist_ok=True)\n",
+    "\n",
+    "onnx_model_path = Path(MODEL_DIR)/onnx_model_path\n",
+    "\n",
+    "!mo --input_model $onnx_model_path --output_dir $MODEL_DIR --model_name $checkpoint --input input_ids,attention_mask --input_shape [1,512],[1,512] --data_type FP32"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "878fbd90",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ir_model_xml = (Path(MODEL_DIR)/checkpoint).with_suffix(\".xml\")\n",
+    "ir_model_bin = (Path(MODEL_DIR)/checkpoint).with_suffix(\".bin\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "27cc074e",
+   "metadata": {},
+   "source": [
+    "Creating the input and output features for the inference"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "026c9eb5",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "BATCH_SIZE = 1\n",
+    "MAX_SEQ_LENGTH = 512\n",
+    "\n",
+    "warnings.filterwarnings(\"ignore\")\n",
+    "\n",
+    "ie = IECore()\n",
```

Comment:
could you please use OpenVINO API 2.0?

Reply:
Yes, modified

Reply:
but looks like not uploaded, I see only 1 commit here for now...

Reply:
yes, I'm testing. I'll upload after making all changes you're asking.